app/views/profiles/show.html.erb

```diff
@@ -62,6 +62,15 @@
               <%= @profile.city %></span>
             </p>
           <% end %>
+
+          <% if @profile.country.present? %>
+            <h2 class="delta sans-serif mb-">
+              <%= t(:country, scope: 'profiles.profile') %>
+            </h2>
+            <p class="bb pb">
+              <%= @profile.country %></span>
```

Comment:
It is still looking the same. You might have to add that to the Profile Model: https://github.com/stefanpenner/country_select#getting-the-country-name-from-the-countries-gem